.github/workflows/release.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     name: Trigger release build
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      contents: read|write
```

Comment:
```suggestion
      contents: 'write'
      id-token: 'write'
      pull-requests: 'write'```

Reply:
☝🏻 this is what I'm using on the website to create PRs

Reply:
Pffff docs smocks... I thought it should be `|` but now I realise they meant OR... :D